Void/Void.Core/Commands.fs

```diff
@@ -3,10 +3,13 @@
 [<RequireQualifiedAccess>]
 type Command =
     | Noop
+    | PublishEvent of Event
+    //
+    | ChangeToMode of Mode

```

Comment:
So you want to call it `NextMode`? This a command, so it should sound like an action, in my opinion...